packages/@cdktf/provider-generator/lib/get/generator/resource-parser.ts

```diff
@@ -35,12 +35,18 @@ const RESERVED_KEYWORDS_FOR_NAMESPACES = [
   "await",
 ];
 
+const COLLIDING_NAMESPACE_NAMES = [
```

Comment:
Is this covered in convert as well?

Reply:
Yes, same reply as here: https://github.com/hashicorp/terraform-cdk/pull/2504#issuecomment-1385688815
😄
Existing follow-up for the "schema-less" convert that does not support this yet is here: https://github.com/hashicorp/terraform-cdk/issues/2508

Reply:
Good good, I just wanted to be sure